### PR TITLE
Add C++YouTube playlist by Naresh i Technologies

### PR DESCRIPTION
**Pull request by sohansai**
## What does this PR do?
Add resource(s)

### Description
This pull request adds a link to the "C++ Programming Language | C++ Tutorial" YouTube playlist  by Naresh i Technologies in the resources section.

### Why is this valuable (or not)?
This is valuable because it provides users with access to a YouTube playlist that can help them learn C++ programming.

### How do we know it's really free?
The resource is a YouTube playlist, which is publicly accessible for free.

### For book lists, is it a book? For course lists, is it a course? etc.
This is a course list, specifically a YouTube playlist.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -342,6 +342,7 @@
 * [C++ Programming](https://www.youtube.com/playlist?list=PLBlnK6fEyqRh6isJ01MBnbNpV3ZsktSyS) - Neso Academy
 * [C++ Programming](https://www.youtube.com/playlist?list=PLfu_Bpi_zcDOtpXhhxL-P1wu0VkwNRXwC) - Kody Simpson
 * [C++ Programming Course - Beginner to Advanced](https://www.youtube.com/watch?v=8jLOx1hD3_o) - Daniel Gakwaya (freeCodeCamp)
+* [C++ Programming Language | C++ Tutorial](https://www.youtube.com/playlist?list=PLVlQHNRLflP8_DGKcMoRw-TYJJALgGu4J) - Naresh i Technologies
```
Comment (eshellman) [inline courses/free-courses-en.md:345]: escape `\`

**Comment by eshellman** [comment]
did you mean to delete two courses?

**Comment by sohansai** [comment]
> did you mean to delete two courses?

No, I meant to add it to the free courses in the repo. It's just one course playlist.

**Comment by eshellman** [comment]
you need to update your branch from main

**Pull request closed without merging**